packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/Diagram.java

```diff
@@ -207,6 +223,7 @@ public Diagram build() {
             diagram.size = Objects.requireNonNull(this.size);
             diagram.nodes = Objects.requireNonNull(this.nodes);
             diagram.edges = Objects.requireNonNull(this.edges);
+            diagram.style = this.style; // Optional on purpose
```

Comment:
The style of a node or an edge are not optional, why would this one be optional?

Reply:
I had assumed that no diagramStyle was created by default, and that only if you wanted a background would you add it.
That's why I made it optional for the different layers.
I'm gonna change that so it's always there and add a participant migration to add it to existing documents.